backend/src/main/java/com/bakdata/conquery/models/auth/entities/User.java

```diff
@@ -181,6 +192,11 @@ public void checkPermissions(Collection<Permission> permissions) throws Authoriz
 			SecurityUtils.getSecurityManager().checkPermissions(getPrincipals(), permissions);
 		}
 
+		@Override
+		public void logout() {
+

```

Comment:
absichtlich leer?

Reply:
kommt wieder raus, das eignet sich eher für einen BackChannelLogout, der aber komplizierter ist, weil man dafür noch ein Logout-Token braucht in OIDC